src/qmflows/parsers/cp2KParser.py

```diff
@@ -57,9 +58,12 @@ def parse_cp2k_warnings(file_name: PathLike,
     return warnings or None
 
 
+#: A generator yielding 2-tuples with warning messages and warning types.
+WarnGenerator = Generator[Tuple[str, Type[Warning]], None, None]
```

Comment:
I don't understand why 2 `None`s.  Would be a single `None` good enough?

Reply:
The [``Generator``](https://docs.python.org/3/library/typing.html#typing.Generator) type is quite picky about its signature.
It requires _exactly_ three values:
1. The ``yield`` type.
2. The ``send`` type.
3. The ``return`` type.

Reply:
I forgot the coroutine part, thanks!